src/engine-actions/is-payment-started.js

```diff
@@ -0,0 +1,41 @@
+const { lookupPaymentStatus } = require('../lnd-actions')
+
+/**
+ * Payment Status corresponding to the output of
+ * LND's LookupPaymentStatus RPC
+ * @constant
+ * @type {Object}
+ */
+const PAYMENT_STATUSES = Object.freeze({
+  GROUNDED: 'GROUNED',
+  IN_FLIGHT: 'IN_FLIGHT',
+  COMPLETE: 'COMPLETE'
+})
+
+/**
+ * Retrieves whether a given payment (as indicated by its hash) has
+ * started. Payments that are currently in-progress or completed are
```

Comment:
It's a little confusing that a complete payment is categorized as a started payment